app/legacy_lib/query_payments.rb

```diff
@@ -302,22 +303,6 @@ def self.create_reverse_select(npo_id, query)
       expr = SearchVector.query(query[:search], expr)
     end
 
-    # This breaks so we need to scrap it
```

Comment:
@wwahammy Also, the code comments for this snippet says that we need to scrap this snippet. 
Is that still true? Do I need to replace it in the file?

Reply:
Let's focus on that in a separate PR, this isn't directly related to the issue.